README.md

```diff
@@ -59,6 +59,24 @@ However, if you wish to run your own static server, there are several options av
 
 Assuming you have Bramble running on port `8000`. Now you can visit [http://localhost:8000/src](http://localhost:8000/src).
 
+
+
+## Keeping up to date
+
+Pull new commits from Mozilla (assuming your mozilla/brackets repo is named 'upstream'). Brackets uses a lot of submodules, which must also be kept up to date.
+```
+git checkout master
+git pull upstream master
+git submodule update --init --recursive
```

Comment:
note that this is technically a super dangerous piece of advice and I don't know if we want to tell people that this is an acceptable solution to a problem that people might not run into. If you just want to keep up to date, you should not need to *have* to do a recursive submodule update.